chore: improve logging for 6.0.x

pgaref opened the pull request:
### Description 
_What behavior do you want to change, why, how does your patch achieve the changes?_

### Testing done 
_Describe the testing strategy. Unit and integration tests are expected for any behavior changes._

### Reviewer checklist
- [ ] Ensure docs are updated if necessary. (eg. if a user visible feature is being added or changed).
- [ ] Ensure relevant issues are linked (description should include text like "Fixes #<issue number>")

vcrfxia reviewed (commented):
Hey @pgaref , 6.0.x introduces a new `/inserts-stream` endpoint which also serves INSERT VALUES requests (in addition to the old `/ksql` endpoint flow that existed on previous versions). We'll have to go through this codepath as well in order to update log messages as needed.

This the handler which gets called: https://github.com/confluentinc/ksql/blob/6.0.x/ksqldb-rest-app/src/main/java/io/confluent/ksql/api/server/InsertsStreamHandler.java
When errors occur, they're forwarded to the acks subscriber: https://github.com/confluentinc/ksql/blob/c47fe8dee3296c76af6ca66316b3b3b6ec64d1f0/ksqldb-rest-app/src/main/java/io/confluent/ksql/api/impl/InsertsSubscriber.java#L161
Which directly logs the error message: https://github.com/confluentinc/ksql/blob/c47fe8dee3296c76af6ca66316b3b3b6ec64d1f0/ksqldb-rest-app/src/main/java/io/confluent/ksql/api/server/AcksSubscriber.java#L106

Which means you'll want to go through the body of `InsertsSubscriber#handleValue()` and vet the error messages which could be thrown from there. I went through most of the pathways and can tell that this error message is a problem: https://github.com/confluentinc/ksql/blob/c47fe8dee3296c76af6ca66316b3b3b6ec64d1f0/ksqldb-rest-app/src/main/java/io/confluent/ksql/api/impl/KeyValueExtractor.java#L85
The only pathway I didn't check in much detail is the serializer pathway: https://github.com/confluentinc/ksql/blob/c47fe8dee3296c76af6ca66316b3b3b6ec64d1f0/ksqldb-rest-app/src/main/java/io/confluent/ksql/api/impl/InsertsSubscriber.java#L146-L147
If you could help check that, that'd be fantastic. 

(BTW, some of these classes exist in 5.5.x but I'm pretty sure we don't need any updates there -- the `/inserts-stream` functionality was incomplete at the time, and the only meaningful error message that could be triggered there (from invalid bytes on the wire) is safe: https://github.com/confluentinc/ksql/blob/d3203ec4a23e37be1c24dce56bed137ca7b0df31/ksqldb-api/src/main/java/io/confluent/ksql/api/server/InsertsStreamHandler.java#L131)

pgaref commented:
> Hey @pgaref , 6.0.x introduces a new `/inserts-stream` endpoint which also serves INSERT VALUES requests (in addition to the old `/ksql` endpoint flow that existed on previous versions). We'll have to go through this codepath as well in order to update log messages as needed.
> 
> This the handler which gets called: https://github.com/confluentinc/ksql/blob/6.0.x/ksqldb-rest-app/src/main/java/io/confluent/ksql/api/server/InsertsStreamHandler.java When errors occur, they're forwarded to the acks subscriber:
> 
> https://github.com/confluentinc/ksql/blob/c47fe8dee3296c76af6ca66316b3b3b6ec64d1f0/ksqldb-rest-app/src/main/java/io/confluent/ksql/api/impl/InsertsSubscriber.java#L161
> 
> 
> Which directly logs the error message:
> https://github.com/confluentinc/ksql/blob/c47fe8dee3296c76af6ca66316b3b3b6ec64d1f0/ksqldb-rest-app/src/main/java/io/confluent/ksql/api/server/AcksSubscriber.java#L106
> 
> Which means you'll want to go through the body of `InsertsSubscriber#handleValue()` and vet the error messages which could be thrown from there. I went through most of the pathways and can tell that this error message is a problem:
> 
> https://github.com/confluentinc/ksql/blob/c47fe8dee3296c76af6ca66316b3b3b6ec64d1f0/ksqldb-rest-app/src/main/java/io/confluent/ksql/api/impl/KeyValueExtractor.java#L85
> 
> 
> The only pathway I didn't check in much detail is the serializer pathway:
> https://github.com/confluentinc/ksql/blob/c47fe8dee3296c76af6ca66316b3b3b6ec64d1f0/ksqldb-rest-app/src/main/java/io/confluent/ksql/api/impl/InsertsSubscriber.java#L146-L147
> 
> 
> If you could help check that, that'd be fantastic.
> (BTW, some of these classes exist in 5.5.x but I'm pretty sure we don't need any updates there -- the `/inserts-stream` functionality was incomplete at the time, and the only meaningful error message that could be triggered there (from invalid bytes on the wire) is safe:
> 
> https://github.com/confluentinc/ksql/blob/d3203ec4a23e37be1c24dce56bed137ca7b0df31/ksqldb-api/src/main/java/io/confluent/ksql/api/server/InsertsStreamHandler.java#L131
> 
> )

Thank you so much for the detailed comment Victoria! Double checked the serializer paths and we are good for all the builtin supported formats.
Latest PR addresses the comments

vcrfxia commented on ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/CommandRunner.java:
```diff
@@ -227,15 +228,17 @@ void fetchAndRunCommands() {
     }
   }
 
+  @SuppressFBWarnings("DMI_INVOKING_TOSTRING_ON_ARRAY")
```
Why is this suppression here? `CommandId` has a `toString()` method and doesn't appear to be an array.

pgaref replied:
I believe the JsonValue is causing the Array Warning here: https://github.com/confluentinc/ksql/blob/954cff98077f98746be40ec6650296b2328e34d0/ksqldb-rest-model/src/main/java/io/confluent/ksql/rest/entity/CommandId.java#L88

This is replaced by getAndDeserializeCommandId() from 6.1.x

vcrfxia reviewed (approved):
Thanks @pgaref -- LGTM with a question inline.
